cmd/oras/internal/option/remote.go

```diff
@@ -121,13 +125,13 @@ func (opts *Remote) Parse() error {
 	if err := opts.parseCustomHeaders(); err != nil {
 		return err
 	}
-	return opts.readPassword()
+	return opts.readPasswordOrIdentityToken()
 }
 
-// readPassword tries to read password with optional cmd prompt.
-func (opts *Remote) readPassword() (err error) {
+// readPasswordOrIdentityToken tries to read password and identity token with optional cmd prompt.
+func (opts *Remote) readPasswordOrIdentityToken() (err error) {
 	if opts.Password != "" {
-		fmt.Fprintln(os.Stderr, "WARNING! Using --password via the CLI is insecure. Use --password-stdin.")
+		fmt.Fprintln(os.Stderr, "WARNING! Using --password or --identity-token via the CLI is insecure. Use --password-stdin and --identity-token-stdin.")
```

Comment:
Should only show `WARNING! Using --identity-token via the CLI is insecure. Use --password-stdin and --identity-token-stdin.` when user name is not set.

Reply:
changed accordingly. See current display screenshot above.